docs/design/autoware-interfaces/ad-api/list/api/routing/index.md

```diff
@@ -22,3 +22,11 @@ There are two ways to set the route. The one is a generic method that uses pose,
 | SET      | The route is set.                                  |
 | ARRIVED  | The vehicle has arrived at the destination.        |
 | CHANGING | Trying to change the route. Not implemented yet.   |
+
+## Goal modification
+
+Autoware may modify the goal pose to pull over, avoid obstacles, etc. When setting a route from the API, applications can choose the following options of goal modification.
```

Comment:
Maybe adding more description might be useful for the reader
```suggestion
Autoware tries to look for an alternate goal when goal is unreachable (e.g., when there is an obstacle on the given goal). When setting a route from the API, applications can choose whether they allow Autoware to adjust goal pose in such situation. When set false, Autoware may get stuck until the given goal becomes reachable.
```

Reply:
Thank you. Applied it.